canonical-web-and-design/maas-ui
# feat(ui): move NUMA resources view from Resources to VMs tab

## Done

- Moved `NumaResources` from `KVMResources` to `LXDHostVMs`

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the "Virtual machines" tab of a LXD single host
- Check that you can toggle by NUMA node view
- navigate away and navigate back and check that the previous selection has persisted
- Go to the "Resources" tab of a LXD single host and check that there is no NUMA node switch

## Fixes

Fixes canonical-web-and-design/app-squad#302
Fixes canonical-web-and-design/app-squad#388
